create_test_database.py

```diff
@@ -15,22 +15,62 @@
 # limitations under the License.
 
 import os
+import time
 
 from google.cloud.spanner_v1 import Client
+from google.cloud.spanner_v1.instance import Instance
+
+
+use_emulator = os.getenv("SPANNER_EMULATOR_HOST") is not None
 
 project = os.getenv(
     "GOOGLE_CLOUD_PROJECT", os.getenv("PROJECT_ID", "emulator-test-project"),
 )
+client = None
```

Comment:
nit: `use_emulator`, `project`, and `client` are effectively global constants so they should be fully capitalized i.e. `USE_EMULATOR`, `PROJECT`, and `CLIENT`

Reply:
Done
Thank you